src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -169,25 +171,39 @@ def delete(self, pool_name):
                                         yes_i_really_really_mean_it=True)
 
     @pool_task('edit', ['{pool_name}'])
-    def set(self, pool_name, flags=None, application_metadata=None, configuration=None, **kwargs):
+    def set(self, pool_name, flags=None, application_metadata=None, configuration=None,
+            mirroring_enabled=None, **kwargs):
```

Comment:
Changed the param's name but we cannot set it to False by default, because we need to disable the mirroing mode as well when the checkbox is unchecked on editing a pool

Reply:
just curious, how this doesn't work with the disabling of mirroring in the edit form?